docs/extend/extend-apiml/onboard-spring-boot-enabler.md

```diff
@@ -111,45 +100,46 @@ Use the following procedure if you use _Maven_ as your build automation system.
 
 **Follow these steps:**
 
-1. Add the following _XML_ tags to your project `pom.xml` file:
-
+1. Add the following _XML_ tags within the newly created `pom.xml` file:
     ```xml
     <repositories>
         <repository>
             <id>libs-release</id>
             <name>libs-release</name>
-            <url>https://zowe.jfrog.io/zowe/libs-snapshot-local/org/zowe/apiml/sdk/onboarding-enabler-spring-v2-springboot-2.1.1.RELEASE/</url>
+            <url>https://zowe.jfrog.io/zowe/libs-release/</url>
             <snapshots>
                 <enabled>false</enabled>
             </snapshots>
         </repository>
     </repositories>
     ```
+   
+    ```
+    **Tip:** If you want to use snapshot version, replace libs-release with libs-snapshot in the repository url and change snapshots->enabled to true.
+
+2. Add the proper dependencies 
+  
+   2.1 For spring version 2, use the following artifact 
+
+   ```maven
+   <dependency>
+       <groupId>org.zowe.apiml.sdk</groupId>
+       <artifactId>onboarding-enabler-spring-v2-springboot-2.1.1.RELEASE</artifactId>
+       <version>$zoweApimlVersion</version>
+   </dependency>
+    ```
 
-2. Create a `settings.xml` file and copy the following _XML_ code block that defines the credentials for the Artifactory:
+   2.2 For spring version 1, use the following artifact 
```

Comment:
Spring version compatibility should be correct
onboarding-enabler-spring-v1-springboot-1.5.9.RELEASE
onboarding-enabler-spring-v2-springboot-2.1.1.RELEASE

Reply:
But these are there.